joe/joe.py

```diff
@@ -40,18 +41,18 @@ def _get_data_dir(path):
 
 def _walk_gitignores():
     '''Recurse over the data directory and return all .gitignore file names'''
-    l = []
-    for root, subFolders, files in os.walk(DATA_DIR):
-        l += [f.replace('.gitignore', '') \
-                for f in files if f.endswith('.gitignore')]
-    return sorted(l)
+    gitignore = []
+    for _, _, files in os.walk(DATA_DIR):
```

Comment:
`root, subFolders` are unused variables. I suppose some information is lost here, I'll add a small commit to alleviate that.